h2o-py/h2o/h2o.py

```diff
@@ -2269,6 +2269,85 @@ def print_mojo(mojo_path, format="json", tree_index=None):
     else:
         raise H2OError("Unable to print MOJO: %s" % output)
 
+def estimate_mem_usage(ncols, nrows, num_cols = 0, string_cols = 0, cat_cols = 0, time_cols = 0, uuid_cols = 0):
+    """
+    Generates and estimate for memory usage in GB.
```

Comment:
I find this to be slightly confusing. We need to keep in mind that memory usage in H2O and required memory to parse a dataset are two different things. Here we are trying to estimate how to size the cluster for a particular dataset. The actual usage will be lower and there will be plenty of memory for munging and modeling building.

IMHO we need to clarify this docustring.

Reply:
I would even rename this method to something like "estimate_cluster_mem" to make it clear it is not estimating memory usage.

Reply:
I have renamed it and updated the documentation.